sysutils/z/Portfile

```diff
@@ -13,8 +14,27 @@ platforms           darwin
 license             WTFPL-2
 supported_archs     noarch
 
-checksums           rmd160  64173ef1b8e77f1db4ee53486f55159ed8b43d83 \
-                    sha256  a06d8d56bd8373229b1f11690ac5d9f13753215ce57488efa1c5a6efc6f735bb
+if {${name} eq ${subport}} {
+   github.setup       rupa z 1.11 v
+   revision 1
+
+   checksums          rmd160  64173ef1b8e77f1db4ee53486f55159ed8b43d83 \
+                      sha256  a06d8d56bd8373229b1f11690ac5d9f13753215ce57488efa1c5a6efc6f735bb
+
+   conflicts          z-devel
+}
+
+subport z-devel {
+   github.setup        rupa z 9d5a3fe0407101e2443499e4b95bca33f7a9a9ca
+   version             20180810
+   revision            0
+
+   checksums           rmd160  bcf75ba6af08a3883f9a70c9a433ae456cfce427 \
+                       sha256  fc82ffe3bbf6535cad08cb7b77481c1e6b1eec974f3c3dd4dbc7a69187e50e36 \
+    size               5940
```

Comment:
The spacing here is strange. Lineup 'size' with the checksums and the value with checksum values.

Reply:
fixed.thx!

Reply:
err... i meant now. maybe :-)
cheers
\--
ferdy